docs/run/instances.mdx

```diff
@@ -43,7 +43,7 @@ When you execute a task using an IBM Quantum service (for example, sending circu
 
 ## Open plan
 
-By default, users who sign up for an IBM Quantum account are assigned to the Open plan and the Open plan's instance, `ibm-q/open/main`. To guarantee that everyone can use the IBM Quantum systems allocated to the plan fairly, **an individual can have no more than three jobs running and/or in the queue (across all systems) at the same time.** Submitting more than three jobs at a time will return error [#3458](../errors#3458), and additional jobs will be canceled.
+By default, users who sign up for an IBM Quantum account are assigned to the Open plan and the Open plan's instance, `ibm-q/open/main`. To guarantee that everyone can use the IBM Quantum systems allocated to the plan fairly, **an individual can have no more than three jobs running and/or in the queue (across all systems) at the same time.** Submitting more than three jobs at a time will return error [#3458](https://quantum-computing.ibm.com/errors#3458), and additional jobs will be canceled.
```

Comment:
Ditto that the original was good. This is a "synthetic file" that we add to the docs when deploying them, even though it's not in this repository. See https://docs.quantum-computing.ibm.com/errors